physics/ground_to_ground_projectile_motion.cpp

```diff
@@ -30,7 +31,7 @@ namespace ground_to_ground_projectile_motion {
  * @param PI The definition of the constant PI
  * @returns Angle in degrees
  */
-double degrees_to_radians(double radian, double PI = 3.14) {
+double degrees_to_radians(double radian, double PI = M_PI) {

```

Comment:
Also isn't there a function in cmath which does this for you?

Reply:
there is a function std::radians(degrees) but it is available only in c++20 version